src/main/scala/jvs/persistence/SchemaRepository.scala

```diff
@@ -46,40 +46,52 @@ object SchemaRepository {
 
   def skunkBased[F[_]: MonadCancelThrow](
     sessionPool: Resource[F, Session[F]]
-  ): SchemaRepository[F] =
-    new SchemaRepository[F] {
+  ): F[SchemaRepository[F]] = {
```

Comment:
changes in this file are only due to the need to initialize tables on first run. In normal circumstances, I'd probably do this at deployment rather than as part of the app.